aws-distro-opentelemetry-node-autoinstrumentation/src/patches/aws/services/bedrock.ts

```diff
@@ -329,13 +348,18 @@ export class BedrockRuntimeServiceExtension implements ServiceExtension {
         if (responseBody.stop_reason !== undefined) {
           span.setAttribute(AwsSpanProcessingUtil.GEN_AI_RESPONSE_FINISH_REASONS, [responseBody.stop_reason]);
         }
-      } else if (currentModelId.includes('cohere.command')) {
-        if (responseBody.prompt !== undefined) {
+      } else if (currentModelId.includes('cohere.command-r')) {
+        console.log('Response Body:', responseBody);
```

Comment:
pls remove all `info` level logging in SDK implementation. we don't want to flood customer's service logs

Reply:
got it. removed this line